walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/MeetingRequest.java

```diff
@@ -19,10 +19,11 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.Set;
 
 public final class MeetingRequest {
   // All the people that should be attending this new meeting. Use a set to avoid duplicates.
-  private final Collection<String> attendees = new HashSet<>();
+  private final Set<String> attendees = new HashSet<>();
```

Comment:
Nice, I think `Set` captures the semantics of this better!

We can't always modify a "public API" like this, but this could also be done in the Query implementation by creating a `new HashSet<>(attendees);` to perform the calculation.